guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -42,6 +43,16 @@
     public static final String PASSWORD_TOKEN = "GUAC_PASSWORD";
 
     /**
+     * The name of the client token added via addStandardTokens().
+     */
+    public static final String REMHOST_TOKEN = "GUAC_REMHOST";
```

Comment:
Yeah, had no problem with it this time around, so probably is was another issue that I confused for the length of the token.  Changed to your suggested values.